diff -urN kactivities-5.18.0/src/service/CMakeLists.txt kactivities-5.18.0-patched/src/service/CMakeLists.txt
--- kactivities-5.18.0/src/service/CMakeLists.txt	2016-01-03 02:22:08.000000000 +1000
+++ kactivities-5.18.0-patched/src/service/CMakeLists.txt	2016-01-12 20:24:27.068864272 +1000
@@ -21,6 +21,7 @@
 set (CMAKE_INCLUDE_CURRENT_DIR ON)
 
 add_library(kactivitymanagerd_plugin SHARED Plugin.cpp Module.cpp Event.cpp Debug.cpp)
+set_target_properties(kactivitymanagerd_plugin PROPERTIES SOVERSION 5)
 generate_export_header(kactivitymanagerd_plugin)
 target_link_libraries(kactivitymanagerd_plugin PUBLIC Qt5::Core Qt5::DBus KF5::CoreAddons KF5::ConfigCore)
 
